lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -89,10 +89,10 @@
             default: 60
             aliases: [ time_out ]
         include_host_uri:
-            description: Toggle to return the additional attribute I(ansible_host_uri) which contains the URI of the
-                         swarm leader in format of M(tcp://172.16.0.1:2376). This value may be used without additional
```

Comment:
`M(xyz)` creates a link to a module called `xyz`, which totally doesn't make sense here.